### ISSUE TITLE
CSS Embedded too tightly in JS code

### ISSUE THREAD

**Issue by nickp-uk** [jackwanders/GalleryView#5]
Would it be possible to externalise more of the CSS into the galleryView CSS file rather than embedding so much into the JS script itself. This would allow easier customisation of the layout - i.e. things like spacing between main image and filmstrip. Appreciate some stuff will still be needed in the JS but think there's way more than is needed at present.

**Comment by S-ed** [comment]
Yep. Same here. I've rewrote code in one project to make it more customizable through css. Maybe later I'll add fork with this code (it can be buggy for some cases so i didn't that yet).
